presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveSchema.java

```diff
@@ -0,0 +1,247 @@
+package io.prestosql.tests.hive;
+
+import com.google.common.collect.ImmutableList;
+import io.prestosql.tempto.AfterTestWithContext;
+import io.prestosql.tempto.BeforeTestWithContext;
+import io.prestosql.tempto.ProductTest;
+import io.prestosql.tempto.assertions.QueryAssert;
+import io.prestosql.tempto.query.QueryResult;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.api.Condition;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.google.common.base.Verify.verify;
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
+import static io.prestosql.tests.utils.QueryExecutors.onPresto;
+import static java.util.Objects.requireNonNull;
+
+public class TestHiveSchema
```

Comment:
Dear Reviewer,
I know that the `TestHiveSchema` is awfully long and looks like 90% redundant, but it is actually not.